examples/kotlin-chat/build.gradle.kts

```diff
@@ -21,7 +21,7 @@ dependencies {
 
 protobuf {
     protoc {
-        artifact = "com.google.protobuf:protoc:3.11.4"
+        artifact = "com.google.protobuf:protoc:3.9.0"
```

Comment:
Would it be interesting to comment on the reason for the downgrade explicitly and what a future update depends on?

Reply:
The current version of `cloudstate-java-support` depends on `com.google.protobuf:protobuf-java-util` and `com.google.protobuf:protobuf-java` libraries version 3.9.0.
It would be nice if we could extract the version of those libraries from dependencies and use it in the build instead of hardcoding it.

Reply:
I agree